tests/config/bootstrap.php

```diff
@@ -63,7 +63,11 @@
 
 if (!getenv('TERMINUS_TESTING_RUNTIME_ENV')) {
     // Create a testing runtime multidev environment.
-    $sitename = TerminusTestTrait::getSiteName();
+    $sitename = (new class {
```

Comment:
There must be a better way to refactor this helper so that we do not need to instantiate an anonymous class just to call the wrapper.

Maybe make a test util class with the useful static methods we need to call from bootstrap. Could move all of the statics from TerminusTestTrait there, optionally with wrappers in TerminusTestTrait, although perhaps this level of cleanup isn't worth the effort.